hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/HtsCallAttempt.java

```diff
@@ -135,6 +144,20 @@ public HtsCallAttempt(@NonNull final Bytes input, @NonNull final HederaWorldUpda
             return MintCall.from(this, senderAddress);
         } else if (BalanceOfCall.matches(selector)) {
             return BalanceOfCall.from(this);
+        } else if (IsOperatorCall.matches(selector)) {
+            return IsOperatorCall.from(this);
+        } else if (TotalSupplyCall.matches(selector)) {
+            return TotalSupplyCall.from(this);
+        } else if (NameCall.matches(selector)) {
```

Comment:
name, symbol, ownerof, tokenuri don't appear in above checklist - also there's both getbalance and balanceof in above checklist.  checklist is useful during this development phase! can you make it match?

Reply:
Fixed list and moved into javadoc segment.